main/src/ca/uwaterloo/flix/api/lsp/provider/completion/ModuleCompleter.scala

```diff
@@ -19,41 +19,14 @@ import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.api.lsp.Index
 import ca.uwaterloo.flix.language.ast.{Symbol, TypedAst}
 import ca.uwaterloo.flix.api.lsp.provider.completion.Completion.ModCompletion
-import ca.uwaterloo.flix.language.ast.Symbol.ModuleSym
+import ca.uwaterloo.flix.api.lsp.provider.completion.CompletionUtils.getNestedModules
 
 object ModuleCompleter extends Completer {
 
   def getCompletions(ctx: CompletionContext)(implicit flix: Flix, index: Index, root: TypedAst.Root, delta: DeltaContext): Iterable[ModCompletion] = {
-    val nestedModules = getNestedModules(ctx)
-    nestedModules.map(mod => ModCompletion(mod))
-  }
 
-  private def getNestedModules(ctx: CompletionContext)(implicit root: TypedAst.Root): List[Symbol.ModuleSym] = {
-    ModuleSymFragment.parseModuleSym(ctx.word) match {
-      case ModuleSymFragment.Complete(modSym) =>
-        root.modules.getOrElse(modSym, Nil).collect {
-          case sym: ModuleSym => sym
-        }
-      case ModuleSymFragment.Partial(modSym, suffix) =>
-        root.modules.getOrElse(modSym, Nil).collect {
-          case sym: ModuleSym if matches(sym, suffix) => sym
-        }
-      case _ => Nil
-    }
+    val nestedModules = getNestedModules(ctx.word)
```

Comment:
We usually don't import functions, so this should be `CompletionUtils.getNestedModules` like in the other file.